tests/tests.h

```diff
@@ -757,35 +757,13 @@ struct SetGrainSizes
 
 DEAL_II_NAMESPACE_CLOSE
 
-/*
- * Do not use a template here to work around an overload resolution issue with
- * clang and enabled  C++11 mode.
```

Comment:
I assume that this statement is outdated, but I can't compile with an old clang version right now: #9672 .